docs/ad/index.md

```diff
@@ -55,12 +55,39 @@ For example, if you choose `min()`, the detector focuses on finding anomalies ba
 You can add a maximum of five features for a detector.
 {: .note }
 
-1. On the **Features** page, select **Add features**.
-1. Enter the **Name** of the feature.
+1. On the **Model configuration** page, enter the **Feature name**.
 1. For **Find anomalies based on**, choose the method to find anomalies. For **Field Value** menu, choose the **field** and the **aggregation method**. Or choose **Custom expression**, and add in your own JSON aggregation query.
-1. Preview sample anomalies and adjust the feature settings if needed.
-- For sample previews, the anomaly detection plugin selects a small number of data samples---for example, one data point every 30 minutes---and uses interpolation to estimate the remaining data points to approximate the actual feature data. It loads this sample dataset into the detector. The detector uses this sample dataset to generate a sample preview of anomaly results.
+
+#### (Optional) Set a category field
+
+You can categorize anomalies based on a keyword or IP field type.
+
+If you specify a category in the same time series but sliced with a different dimension like IP addresses, product IDs, country codes, and so on, you’ll see a granular view of anomalies within each entity of that field. This helps to dive deeper into anomalies of a unique entity or ID and isolate and debug issues.
+
+To set a category field, choose **Enable a category field** and select a field.
+
+Only a certain number of unique entities are supported in the category field. Use the following equation to calculate the recommended total number of entities number supported in a cluster:
+
+```
+(JvmHeapSizeInMb / 20) * (DataNodesCount)
```

Comment:
@weicongs-amazon @kaituo, can you guys comment on this formula? does this mean each entity takes 20M memory?

Reply:
I thought we cannot have a formula due to the query size limit. 

Ignoring the query size limit, this is the formula:

```
# data nodes * heap size * AD memory max percentage / entity size of an detector
```

AD memory max percentage is 10% by default (adjustable via ES setting).  We can fetch entity size of an detector using profile API.

For example, for a cluster with 3 data nodes, each with 8G of JVM heap size, AD memory max percentage is 10% (default value) , entity size of the detector is 1MB,  the total number of unique entities supported is (8.096  * 10^ 9 *0.1 / 1M  ) * 3 = 2429

Reply:
Added to the docs. Please review.